### PR TITLE
UI - strata + platform dashbaord updates

### PR DESCRIPTION
**Pull request by kialj876**
*Issue:*

- bcgov/entity/issues/

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-286-72dg91y8.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-286-lmhgh3ag.web.app

File: strr-base-web/app/composables/useStrrApi.ts [hunk 1]
```diff
@@ -5,7 +5,10 @@ export const useStrrApi = () => {
   const getAccountRegistrations = async <T extends ApiBaseRegistration>(id?: string) => {
     // TODO: add optional filter based on registration type (need in api first)
     if (id) {
-      return await $strrApi<T>(`/registrations/${id}`)
+      return await $strrApi<T>(`/registrations/${id}`).catch((e) => {
+        logFetchError(e, `Unable to get registration details for ${id}`)
```
Comment (deetz99) [inline strr-base-web/app/composables/useStrrApi.ts:9]: 👍

File: strr-base-web/app/utils/date.ts
Comment (deetz99) [inline strr-base-web/app/utils/date.ts:1]: not urgent but we should update this to use [luxon](https://github.com/moment/luxon) eventually

**Review by deetz99 — APPROVED** [review]
All good assuming the application/registration GETS sort themselves out once deployed 👍

**Pull request merged**